recipes/hazelcast-cpp-client/all/conanfile.py

```diff
@@ -75,10 +74,10 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
 
     def package_info(self):
-        self.cpp_info.filenames["cmake_find_package"] = "hazelcastcxx"
-        self.cpp_info.filenames["cmake_find_package_multi"] = "hazelcastcxx"
-        self.cpp_info.names["cmake_find_package"] = "hazelcastcxx"
-        self.cpp_info.names["cmake_find_package_multi"] = "hazelcastcxx"
+        self.cpp_info.filenames["cmake_find_package"] = "hazelcast-cpp-client"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "hazelcast-cpp-client"
+        self.cpp_info.names["cmake_find_package"] = "hazelcast-cpp-client"
+        self.cpp_info.names["cmake_find_package_multi"] = "hazelcast-cpp-client"
```

Comment:
Since this is the only real change you can have one recipe for both versions with

```py
@property
def _cmake_name(self):
    return "hazelcastcxx" if tools.Version(self.version) <= "4.0.0" else "hazelcast-cpp-client"
```

In the CMake you can do something similar

Reply:
fixed.